### ISSUE TITLE
Ceph: CrashCollector pod CrashLoopBackOff due to permission denied on /var/lib/ceph/crash

### ISSUE THREAD

**Issue by ron1** [rook/rook#5031]
**Bug Report**
CrashCollector pod CrashLoopBackOff due to permission denied on /var/lib/ceph/crash

**Deviation from expected behavior:**
CrashCollector pod experiences CrashLoopBackOff with the following log:
```
INFO: __main__:monitoring path /var/lib/ceph/crash, delay 600s
traceback (most recent call last):
  File "/usr/bin/ceph-crash", line 98, in <module>
    main()
  File "/usr/bin/ceph-crash", line 91, in main
    scrape_path(args.path)
  File "/usr/bin/ceph-crash", line 58, in scrape_path
    for p in os.listdir(path):
OSError: [Errno 13] Permission denied: '/var/lib/ceph/crash'
```

**Expected behavior:**
CrashCollector pod deploys successfully

**How to reproduce it (minimal and precise):**
Install rook-ceph v1.2.6 on OCP 3.11

I was able to correct the problem by scaling down the rook-ceph-operator pod and patching the ceph-crash container of the rook-ceph-crashcollector deployment to include a privileged securityContext.

**Environment**:
* OS (e.g. from /etc/os-release): RHEL 7.7
* Rook version (use `rook version` inside of a Rook Pod): rook-ceph v1.2.6
* Storage backend version (e.g. for ceph do `ceph -v`): v14.2.8
* Kubernetes version (use `kubectl version`): 1.11+
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift): OpenShift 3.11

**Comment by leseb** [comment]
@rohan47 please look into this once you're done with the restart/node gone issue. Thanks!

**Comment by ron1** [comment]
Pls reopen

**Comment by leseb** [comment]
I think we fixed that something in 1.3 or master...
@ron1 still seeing that issue?

**Comment by travisn** [comment]
@ron1 Please do reopen if you are still seeing the issue, but it's expected to be fixed in the latest 1.3 patch release by #5548.

**Comment by ron1** [comment]
Thanks. I'll verify and reopen if needed.
